src/cleanandgo/Equipment.java

```diff
@@ -70,35 +70,83 @@ private static void totalNumberOfEquipment() {
-		// TODO need to implement this by #3
-		System.out.println("maintenanceSchedule not yet implemented...");
-	}
+        System.out.println("---------------------------------------------------------");
+        System.out.println("       Weekly Maintainance Schedule for Equipment           ");
+        System.out.println("---------------------------------------------------------");
+        Connection conn = DBConnection.getConnected();
+        try {
+            do {
+                String query = "SELECT * FROM Maintainance WHERE date BETWEEN DATE_ADD(now(), interval -7 DAY) and now() "
+                + "AND Equipment_idEquipment = ?";
+                PreparedStatement p = conn.prepareStatement(query);
+                String EquipmentId = Util.getUsersInput("Type in the equipment id: ");
+                p.clearParameters();
+                p.setString(1, EquipmentId);
+                ResultSet rset = p.executeQuery();
+                while (rset.next()) {
+                    int idEquipment = rset.getInt(1);
+                    int idEmployee = rset.getInt(2);
+                    java.sql.Date date = rset.getDate(3);
+                    Double cost = rset.getDouble(4);
+                    String descr = rset.getString(5);
+                    System.out.println("idEquipment: " + idEquipment);
+                    System.out.println("idEmployee: " + idEmployee);
+                    System.out.println("Date: " + date);
+                    System.out.println("Cost: " + cost);
+                    System.out.println("Description: " + descr + "\n");
+                    
+                }
+            } while (!Util.getUsersInput("Type X to exit or any button to search for another schedule: ").equals("X"));
+        } catch (SQLException e) {
+            System.out.println("---------------------------------------------------------");
+            System.out.println("Looks like an equipment with this id doesn't exist");
+            System.out.println(e);
+            System.out.println("---------------------------------------------------------");
+        }
+    }
 
 	/**
      * Prints a report for the average daily usage, for each equipment,
      *  during the current year
      *  */
-    public static void printEquipmentUsageAnualReport() {
+    public static void averageDailyUsage() {
         System.out.println("---------------------------------------------------------");
         System.out.println("  Average daily usage of equipment for the current year  ");
         System.out.println("---------------------------------------------------------");
         Connection conn = DBConnection.getConnected();
         try {
             do {
-                String query = "to be implementated";
+                //currently this queries shows all dates in a year, per equipment
+                String query = "SELECT  S.Equipment_idEquipment, R.service_date FROM Customer_Receives_Service AS R, Equipment_has_Service AS S " + 
```

Comment:
Not sure about this query. It seems like it's printing equipmentID and service_date, but nothing really about the average daily usage time. I feel we would need to rewrite it again after we will change the DB design

Reply:
Yes, this was just me attempting to do the query. I seems very advanced lol. I will remove it